csunplugged/topics/management/commands/_ProgrammingExercisesLoader.py

```diff
@@ -94,6 +105,7 @@ def load(self):
                 exercise_set_number=exercise_set_number,
                 exercise_number=exercise_number,
                 content=exercise_content.html_string,
+                extra_challenge=exercise_extra_challenge,
```

Comment:
to be consistent, shouldn't pull out the html_string before this step, should just do it all in one go (like 107), i.e. `extra_challenge=exercise_extra_challenge_content.html_string`